pkg/indexmanagement/scripts.go

```diff
@@ -50,8 +56,8 @@ r=json.load(sys.stdin)
 indices = [index for index in r if int(r[index]['settings']['index']['creation_date']) < $minAgeFromEpoc ]
 if "$writeIndex" in indices:
   indices.remove("$writeIndex")
-indices.sort()
-print ','.join(indices)
+for i in range(0, len(indices), 25):
+  print ','.join(indices[i:i+25]) + " "
```

Comment:
doesn't `', '.join(` yield you the same? the trailing `" "` looks odd to me here

Reply:
probably, experimenting in a debug pod wasn't fun so did what i could.  ill update it